### DISCUSSION TITLE
Is usage of pybel allowed?

### DISCUSSION THREAD

**Discussion by sooheon** [snap-stanford/ogb#190] [PCQM4M-LSC]
AFAIK pybel is just convenience wrappers around Open Babel, and the latter is explicitly allowed in the rules.

**Comment by weihua916** [comment]
Yes, you can use pybel.

**Comment by sooheon** [comment]
Is it intended behavior that not all molecules may be conformed/optimized with rdkit/openbabel with default settings?

**Reply by weihua916** [reply]
Yes. The 3D molecular geometry in the PCQM4M-LSC dataset is optimized with DFT, which is far more accurate (and expensive) than rdkit's geometry optimization.
